test/cppsuite/src/component/timestamp_manager.cpp

```diff
@@ -133,10 +133,21 @@ timestamp_manager::get_oldest_ts() const
 }
 
 wt_timestamp_t
-timestamp_manager::get_random_ts() const
+timestamp_manager::get_valid_read_ts() const
 {
+    /* Use get_oldest_ts here to convert from atomic to wt_timestamp_t. */
+    wt_timestamp_t current_oldest = get_oldest_ts();
+    wt_timestamp_t current_stable = _stable_ts;
+    if (current_stable > current_oldest) {
+        --current_stable;
+    }
+    testutil_assert(current_stable > 0 && current_stable >= current_oldest);
```

Comment:
It will always be positive as it's an unsigned. And I think the second part should always be guaranteed, doesn't hurt to leave it though.

Reply:
Fixed in [d469706](https://github.com/wiredtiger/wiredtiger/pull/7985/commits/d4697067a92d10fc49f934518be3685551af3b47)